pages/posts/[id].js

```diff
@@ -0,0 +1,43 @@
+import Page from '../../components/base_page';
+import { getAllPostIds, getPostData } from '../../components/posts_data';
+import Date from '../../components/date';
+
+import css from '../../styles/common.module.scss';
+
+import Head from 'next/head';
+
+export default function Post({ postData }) {
```

Comment:
How will the URL appear? For SEO (https://en.wikipedia.org/wiki/Search_engine_optimization) purposes, it's important that the URL be human-readable and not merely a bunch of numbers.

Reply:
The URL is www.aweandreverence.com/posts/test